src/transformers/modeling_transfo_xl.py

```diff
@@ -658,6 +658,11 @@ class TransfoXLLMHeadModelOutput(ModelOutput):
     hidden_states: Optional[Tuple[torch.FloatTensor]] = None
     attentions: Optional[Tuple[torch.FloatTensor]] = None
 
+    @property
+    def logits(self):
```

Comment:
@sgugger - this is needed for generation.

Reply:
Note that they are not logits but went through the adaptive softmax, not sure if it breaks things for you or not. Should probably be documented as a warning.

Reply:
Yeah, I see what you mean. I make the comment clearer. 
Given that `predicition_scores` is a logsoftmax, it behaves more or less exactly like "logits" (The lower, the less likely to be sampled, the higher the more likely to be sampled), so IMO it's fine to have the property here.